test/vscodeLauncher.ts

```diff
@@ -0,0 +1,88 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as cp from 'child_process';
```

Comment:
this was split from the old integration test runner.  We actually only need 1 copy of this, and not one for each jest integration test project (cc @davidwengier ).  But we do still need 1 index per jest project